src/app/api/member/[username]/route.ts

```diff
@@ -99,6 +104,23 @@ export async function PUT(
     });
 }
 
+/**
+ * Get member with given api key
+ */
+export async function GET(
+    req: NextRequest,
+    { params: { username } }: { params: { username: string } }
+) {
+    const invalidTokenResponse = validateApiToken(req);
```

Comment:
wording: what about `getInvalidTokenResponse` ? `validateApiToken` that return an invalideResponse sounds misleading